### PR TITLE
feat: add Sprint 32 batch

### PR DESCRIPTION
**Pull request by avious00**
This subset will be core deployed + relayed: [[discord convo]](https://discord.com/channels/935678348330434570/1275532188380561439/1275863462655819818)
- Aleph Zero
- Astar
- Astar zkEVM - keys are funded
- Bitlayer - funded
- Dogechain
- Flare - funded
- Shib
- Core - needs alignment with their team, work directly with them in telegram
- Oort - needs to be migrated, work directly with them in telegram (metadata not included)
- Molten/Unidex -  metadata is in another PR https://github.com/hyperlane-xyz/hyperlane-registry/pull/147/files (funds sent)

Funding by Soorya can be [tracked in Notion](https://www.notion.so/hyperlanexyz/Fund-Sprint-32-Deployment-List-ec07f139fa2448b68a4bbf6c9de86a5d)

**Notes:**
- NGMI chainlet is a EVM rollup by Saga on Cosmos; Saga devrel confirmed this is supported by our CLI. Keys are funded already https://medium.com/sagaxyz/stack-explorer-bridging-your-saga-chainlet-assets-across-evm-networks-using-hyperlane-28b1cfe8f69e
- conflux VM diff https://doc.confluxnetwork.org/docs/core/core-space-basics/vm-difference
- cronos zkevm gas fixed to 2500 gwei https://docs-zkevm.cronos.org/for-developers/resources-for-developers#cronos-zkevm-mainnet-urls
- Rari keys are funded already 

**Original List:** Aleph Zero, Arbitrum Nova, Astar, Astar zkEVM, B3, Bitlayer, Carbon, Clique, Conflux CoreSpace, Conflux eSpace, Core, Cronos, Cronos zkEVM, Dogechain, Ebi, Filecoin, Flare, Fractal, Funki, Ham, Kava, Molten, NGMI chainlet, OORT, Orderly, Polynomial, Pulsechain, Rari, Ronin, Saakuru, Shibarium, and Tenet

### CODE REVIEW

**Comment by avious00** [comment]
@paulbalaji based on your feedback going to add to this gradually as opposed to all at once

File: chains/astar/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+blockExplorers:
+  - apiUrl: https://astar.subscan.io/api-docs
+    family: subscan
```
Comment (jmrossy) [inline chains/astar/metadata.yaml:3]: the valid values here are etherscan | blockscout | other

File: chains/astar/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    name: Astar Explorer
+    url: https://astar.subscan.io
```
Comment (paulbalaji) [inline chains/astar/metadata.yaml:5]:
```suggestion
  - apiUrl: https://astar.blockscout.com/api
    family: blockscout
    name: Astar Explorer
    url: https://astar.blockscout.com
```

**Review by paulbalaji — COMMENTED** [review]
are we adding astar or the astar zkevm?

**Comment by avious00** [comment]
> are we adding astar or the astar zkevm?

both, more commits coming

File: chains/cronoszkevm/metadata.yaml
Comment (paulbalaji) [inline chains/cronoszkevm/metadata.yaml:1]: not to self:
image

File: chains/fractal/logo.svg
Comment (paulbalaji) [inline chains/fractal/logo.svg:1]: i think this logo is for the wrong fractal?

File: chains/fractal/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+blocks:
+  confirmations: 1
+  estimateBlockTime: 2
+  reorgPeriod: 0
+chainId: 12788
+deployer:
+  name: Abacus Works
+  url: https://www.hyperlane.xyz
+displayName: Fractal Confluence
+domainId: 12788
```
Comment (paulbalaji) [inline chains/fractal/metadata.yaml:15]: had to change ID to 2152 I think, for fractal confluence https://docs.fra.tech/network-settings/network-settings

File: chains/saakuru/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+blocks:
+  confirmations: 1
+  estimateBlockTime: 2
+  reorgPeriod: 0
+chainId: 24734
+deployer:
+  name: Abacus Works
+  url: https://www.hyperlane.xyz
+displayName: Saakuru
+domainId: 24734
```
Comment (paulbalaji) [inline chains/saakuru/metadata.yaml:15]: this chain id was also wrong? should be 7225878

Comment (avious00) [reply chains/saakuru/metadata.yaml]: @paulbalaji sorry for missing these, thanks for catching

File: chains/alephzero/metadata.yaml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+domainId: 787
+gasCurrencyCoinGeckoId: aleph-zero
+name: alephzero
+nativeToken:
+  decimals: 18
+  name: AZERO
+  symbol: AZERO
+protocol: ethereum
+rpcUrls:
+  - http: https://rpc.alephzero.raas.gelato.cloud
```
Comment (jmrossy) [inline chains/alephzero/metadata.yaml:24]: I'm seeing single RPC URLs for a lot of these. Any chance we can find other public ones?

File: chains/bitlayer/metadata.yaml [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+  name: Abacus Works
+  url: https://www.hyperlane.xyz
+displayName: Bitlayer
+domainId: 200901
+gasCurrencyCoinGeckoId: bitcoin
+name: bitlayer
+nativeToken:
+  decimals: 18
+  name: Bitcoin
+  symbol: BTC
```
Comment (jmrossy) [inline chains/bitlayer/metadata.yaml:21]: This may be the first BTC in the registry, cool to see it

File: chains/filecoin/metadata.yaml [hunk 7]
```diff
@@ -0,0 +1,25 @@
+blockExplorers:
+  - apiUrl: https://filecoin.blockscout.com/api
+    family: blockscout
+    name: Filecoin Virtual Machine Explorer
```
Comment (jmrossy) [inline chains/filecoin/metadata.yaml:4]:
```suggestion
    name: Filecoin VM Explorer
```

File: chains/funki/metadata.yaml [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+  estimateBlockTime: 2
+  reorgPeriod: 0
+chainId: 33979
+deployer:
+  name: Abacus Works
+  url: https://www.hyperlane.xyz
+displayName: Funki
+domainId: 33979
+gasCurrencyCoinGeckoId: ethereum
+gnosisSafeTransactionServiceUrl:
```
Comment (jmrossy) [inline chains/funki/metadata.yaml:17]: Let's set it or exclude it

File: chains/ngmi/logo.svg
Comment (jmrossy) [inline chains/ngmi/logo.svg:1]: Consider removing the black background here, but not mandatory

File: chains/ngmi/metadata.yaml [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+  estimateBlockTime: 2
+  reorgPeriod: 0
+chainId: 2712765173321000
+deployer:
+  name: Abacus Works
+  url: https://www.hyperlane.xyz
+displayName: NGMI
+domainId: 2712765173321000
+gasCurrencyCoinGeckoId: ngmi
+gnosisSafeTransactionServiceUrl:
```
Comment (jmrossy) [inline chains/ngmi/metadata.yaml:17]: ditto

**Review by jmrossy — COMMENTED** [review]
Wow, lots of chains! This is coming along well but still missing some logos (as you probably know)

File: chains/bitlayer/metadata.yaml (hunk 6 above)
Comment (paulbalaji) [reply chains/bitlayer/metadata.yaml:21]: second I think - merlin might be the first https://github.com/hyperlane-xyz/hyperlane-registry/blob/main/chains/merlin/metadata.yaml#L16

File: chains/alephzero/metadata.yaml (hunk 5 above)
Comment (paulbalaji) [reply chains/alephzero/metadata.yaml:24]: I tried finding some more for them in my first pass, was going to do another pass on rpc urls when the set of chains gets finalised

File: chains/ngmi/metadata.yaml [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+  confirmations: 1
+  estimateBlockTime: 2
+  reorgPeriod: 0
+chainId: 2712765173321000
+deployer:
+  name: Abacus Works
+  url: https://www.hyperlane.xyz
+displayName: NGMI
+domainId: 2712765173321000
+gasCurrencyCoinGeckoId: ngmi
```
Comment (paulbalaji) [inline chains/ngmi/metadata.yaml:16]: double checking this is the right gecko entry? https://www.coingecko.com/en/coins/ngmi

File: chains/confluxcore/metadata.yaml [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+name: confluxcore
+nativeToken:
+  decimals: 18
+  name: Conflux
+  symbol: CFX
+protocol: ethereum
+rpcUrls:
+  - http: https://main.confluxrpc.com
+  - http: https://mainnet-rpc.confluxnetwork.org
+  - http: https://core.confluxrpc.com
```
Comment (paulbalaji) [inline chains/confluxcore/metadata.yaml:28]: these two RPCs don't work, removing

File: chains/confluxespace/metadata.yaml [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+name: confluxespace
+nativeToken:
+  decimals: 18
+  name: Ethereum
+  symbol: ETH
+protocol: ethereum
+rpcUrls:
+  - http: https://evm.confluxrpc.com
+  - http: https://conflux-espace.publicnode.com
+  - http: https://rpc.confluxnetwork.org/espace
```
Comment (paulbalaji) [inline chains/confluxespace/metadata.yaml:27]: these two rpcs don't work, removing

File: chains/funki/metadata.yaml [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+gasCurrencyCoinGeckoId: ethereum
+name: funki
+nativeToken:
+  decimals: 18
+  name: Ethereum
+  symbol: ETH
+protocol: ethereum
+rpcUrls:
+  - http: https://rpc.funkichain.com
+  - http: https://rpc.thirdweb.com/funki
```
Comment (paulbalaji) [inline chains/funki/metadata.yaml:25]: neither work, changing to `https://rpc-mainnet.funkichain.com`

File: chains/ngmi/metadata.yaml [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+name: ngmi
+nativeToken:
+  decimals: 18
+  name: NGMI
+  symbol: NGMI
+protocol: ethereum
+rpcUrls:
+  - http: https://rpc.saga.xyz/chainlets/ngmi_2712765173321000-1
```
Comment (paulbalaji) [inline chains/ngmi/metadata.yaml:24]: this rpc doesn't work, can't find alternatives either

is there a docs page for this chain?

**Comment by paulbalaji** [comment]
Couple comments:
- a quick way to test RPC urls is by doing `cast bn --rpc-url <rpcUrl>`, and it should return a block number
- it would be great going forward to add the logos as part of the metadata addition, so we can keep the CI green as we add chains in the branch

File: chains/ngmi/metadata.yaml (hunk 14 above)
Comment (paulbalaji) [reply chains/ngmi/metadata.yaml:24]: found https://app.saga.xyz/chainlets/ngmi_2712765173321000-1

File: chains/ngmi/metadata.yaml (hunk 10 above)
Comment (avious00) [reply chains/ngmi/metadata.yaml:16]: that doesnt look right :/ added you to the dev chat (Land Labor Capitol)

Comment (avious00) [reply chains/ngmi/metadata.yaml:16]: no coingeckoid yet

**Review by paulbalaji — APPROVED** [review]

**Pull request merged**